src/ray/core_worker/actor_handle.h

```diff
@@ -24,7 +24,7 @@
 #include "src/ray/protobuf/gcs.pb.h"
 
 namespace ray {
-
+namespace core {
```

Comment:
This PR looks great overall. The only concern I have is that whether we should rename `ray` to `ray::core` as a whole.

Reply:
+1